suitcase/hdf5.py

```diff
@@ -2,21 +2,32 @@
 import warnings
 import h5py
 import json
+import copy
 from metadatastore.commands import get_events_generator
 from databroker.databroker import fill_event
 from databroker.core import Header
 
 
 __version__ = "0.2.2"
 
-def export(headers, filename):
+def export(headers, filename, stream_name=None, fields_unwanted=None, timestamps_opt=True):
     """
+    Create hdf5 file to preserve the structure of databroker. Necessary copy are made at each step, so
+    headers will not be modified.
+
     Parameters
     ----------
     headers : a Header or a list of Headers
         objects retruned by the Data Broker
     filename : string
         path to a new or existing HDF5 file
+    stream_name : string, optional
+        None means save all the data from each descriptor, i.e., user can define stream_name as primary,
+        so only data with descriptor.name == primary will be saved.
+    fields_unwanted : list, optional
+        list of names which are excluded when data is transfered to HDF5 file
+    timestamps_opt : Bool, optional
```

Comment:
drop the `_opt`. I'd like to make sure that we differentiate between `Event.time` and `Event.timestamps`, 
with the former being the bundling time of the Event and the latter being the individual EPICS timestamps
of all the signals that are recorded in the Event.  This kwarg, IIUC, is to *not* include `Event.timestamps`.
As such, I'd use something like the following to describe this kwarg:


```
timestamps : bool, optional
    True/False: Do/Don't export the EPICS timestamps of every signal captured.  Note that 
    the events also have a time field (Event.time) that indicates the time when the client 
    captured all relevent information and created the `Event` document. This kwarg disables
    exporting the `Event.timestamps` data which is the EPICS timestamp of every signal captured.
```